src/countOccurrences.js

```diff
@@ -18,7 +18,9 @@
  *
  */
 function countOccurrences(phrase, letter) {
-  // write code here
+  const reg = new RegExp(letter, 'gi');
+  const coinsidenceArr = phrase.match(reg);
+  return coinsidenceArr ? coinsidenceArr.length : 0;
```

Comment:
new line
```suggestion

  return coinsidenceArr ? coinsidenceArr.length : 0;
```
also you have mistake `coinSidenceArr` => `coinCidenceArr`

Reply:
Every time I should add empty line before return? 
Coinsidence - its one word, I should write this word in CamelCase?

Reply:
yes

Reply:
you can look here - https://github.com/mate-academy/style-guides/blob/master/javascript.md#whitespace

Reply:
Thank you, I will read this style guide.